export const catching = () => {
  return (Math.floor(Math.random() * 2)) ? 'success' : 'failed';
}